src/features/CodeActions.ts

```diff
@@ -23,13 +25,34 @@ export class CodeActionsFeature implements IFeature {
                     edit.Text);
             });
         });
+
+        this.showDocumentationCommand =
+            vscode.commands.registerCommand("PowerShell.ShowCodeActionDocumentation", (ruleName: any) => {
+                this.showRuleDocumentation(ruleName);
+            });
     }
 
     public dispose() {
-        this.command.dispose();
+        this.applyEditsCommand.dispose();
+        this.showDocumentationCommand.dispose();
     }
 
     public setLanguageClient(languageclient: LanguageClient) {
         this.languageClient = languageclient;
     }
+
+    public showRuleDocumentation(ruleId: string) {
+        const pssaDocBaseURL = "https://github.com/PowerShell/PSScriptAnalyzer/blob/development/RuleDocumentation";
+
+        if (!ruleId) {
+            this.log.writeWarning("Cannot show documentation for code action, no ruleName was supplied.");
+            return;
+        }
+
+        if (ruleId.startsWith("PS")) {
+            ruleId = ruleId.substr(2);
```

Comment:
Damn that's annoying

Reply:
Yeah, it's one of those things that on the [rule docs page](https://github.com/PowerShell/PSScriptAnalyzer/tree/development/RuleDocumentation) having every topic header start with `PS` makes it a bit harder to find a rule.  Well, I "believe" that was the motivation for the naming there.  :-)